scriptworker/constants.py

```diff
@@ -255,6 +255,18 @@
                 'project:releng:treescript:action:push': 'all-release-branches',
             }),
             'thunderbird': frozendict({
+                'project:comm:thunderbird:releng:balrog:server:nightly': 'all-nightly-branches',
+                'project:comm:thunderbird:releng:balrog:server:beta': 'beta',
+                'project:comm:thunderbird:releng:balrog:server:esr': 'esr',
+
+                'project:comm:thunderbird:releng:beetmover:bucket:nightly': 'all-nightly-branches',
+                'project:comm:thunderbird:releng:beetmover:bucket:release': 'all-release-branches',
+
+                'project:comm:thunderbird:releng:bouncer:server:production': 'all-production-branches',
```

Comment:
I'm not sure if `all-production-branches` is clear - we run a mixed production/staging env on maple for Firefox, for instance. Release signing with staging balrog/beetmover/bouncer/etc. Right now `all-production-branches` appears to match `all-release-branches`.